crypto/asn1/a_mbstr.c

```diff
@@ -272,12 +270,15 @@ static int out_utf8(unsigned long value, void *arg)
 static int type_str(unsigned long value, void *arg)
 {
     unsigned long types;
+    const int native = ossl_fromascii(value);
```

Comment:
I think can do this with a macro I don't like:
`#define fromascii(c) ((c) != (int)(c) ? (c) : ossl_fromascii(c))`
Plus others for the ossl_isXXX functions etc.

I really don't this idea because of the multiple occurrences of the argument in the macro.  Side effects would break it completely.